spec/sqewer/active_job_spec.rb

```diff
@@ -92,4 +130,46 @@ class User < ActiveRecord::Base
     
     wait_for { user.reload.active? }.to eq(true)
   end
+
+  it 'Create file by options given with option arguments' do
```

Comment:
Spec titles are usually transformed context-to-example and they tend to read nicer if they conform to `it "does ..."`. "It saves", "it quits", "it removes" and so forth. This is also how most of our repos do it.